app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden:
+            self.hidden = False
+        elif not self.hidden:
+            self.hidden = True
```

Comment:
You can shorten if-elif condtion in on line by giving self.hidden it's opposite value using not operator: `self.hidden = not self.hidden`